taurus/demo/cake.py

```diff
@@ -123,6 +123,11 @@ def make_cake_templates():
         description="Sample size in mass units, to go along with FDA Nutrition Facts",
         bounds=RealBounds(1.e-3, 10.e3, "g")
     )
+    tmpl["Expected Sample Size"] = ParameterTemplate(
+        name="Expected Sample Size",
```

Comment:
"Expected Sample Size" sounds like it is the expected number of samples. Why not "Expected Sample Mass"?